src/Core/TileGeometry.js

```diff
@@ -35,24 +35,13 @@ function TileGeometry(params, builder) {
     this.center = builder.Center(params).clone();
     this.OBB = builder.OBB(params);
 
-    // TODO : free array
-
-    var bufferAttribs = this.computeBuffers(params, builder);
+    const bufferAttribs = this.computeBuffers(params, builder);
 
     this.setIndex(bufferAttribs.index);
-    this.addAttribute('position', bufferAttribs.position);
-    this.addAttribute('normal', bufferAttribs.normal);
+    this.addAttribute('position', new THREE.BufferAttribute(bufferAttribs.position, 3));
+    this.addAttribute('normal', new THREE.BufferAttribute(bufferAttribs.normal, 3));
     this.addAttribute('uv_wgs84', bufferAttribs.uv.wgs84);
-    this.addAttribute('uv_pm', bufferAttribs.uv.pm);
-
-    bufferAttribs.position = null;
-    bufferAttribs.normal = null;
-    bufferAttribs.uv.pm = null;
-
-    // Update cache
-    if (!cache.getRessource(params.segment)) {
-        cache.addRessource(params.segment, bufferAttribs);
-    }
+    this.addAttribute('uv_pm', new THREE.BufferAttribute(bufferAttribs.uv.pm, 1));
```

Comment:
I would suggest having either everything as BufferAttributes, or everything as TyperArray in this `bufferAttribs`.